index.html

```diff
@@ -8,5 +8,5 @@
 <body>
     <div id="root"></div>
 </body>
-<script src="/src/index.tsx"></script>
+<script type="module" src="./src/index.tsx"></script>
```

Comment:
Hey @carlota-moh 
I've been checking your reported issue and I have a question:
Why did you add this line to load the `./src/index.tsx`?

Try just removing this line and let's see if your issue is fixed, let me know what happens, thanks.

Reply:
Hi @aemartinez37 
Thanks for the input!! I suspected it had something to do with that, although I couldn't quite figure out why. I researched into it and I realized that the webpack HTML plugin was the one in charge of loading the compiled index into the HTML. Since browsers can't read TS, it was being interpreted as a JSON rather than an executable and it obviously could not locate it... Dumb mistake on my part, thanks for helping :)